### PR TITLE
Added task for generating custom certs

### PR DESCRIPTION
**Pull request by ntkathole**

### CODE REVIEW

File: upgrade/helpers/tasks.py [hunk 1]
```diff
@@ -799,3 +799,33 @@ def pre_upgrade_system_checks(capsules):
     if capsules:
         for capsule in capsules:
             wait_untill_capsule_sync(capsule)
+
+
+def generate_custom_certs():
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: I think, This task is more aligned with https://github.com/SatelliteQE/automation-tools than https://github.com/SatelliteQE/satellite6-upgrade

Comment (ntkathole) [reply upgrade/helpers/tasks.py]: @jyejare I have a scenario where I need some tasks from automation tools and some upgrade related task from satellite6-upgrade. Satellite6-upgrade has requirement of automation tools but automation-tools don't have satellite6-upgrade dependency, so I can easily clone satellite6-upgrade and use required automation tools task to achieve it. 
https://github.com/SatelliteQE/satellite6-upgrade/blob/master/requirements.txt#L2

File: upgrade/helpers/tasks.py [hunk 2]
```diff
@@ -799,3 +799,33 @@ def pre_upgrade_system_checks(capsules):
...
+    """ Task to generate custom certs for satellite
+    :param: SERVER_HOSTNAME
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: There is no param as such, modify this as:

```suggestion
    environment variables:
    
    SERVER_HOSTNAME The satellite server hostname
```

File: upgrade/helpers/tasks.py [hunk 3]
```diff
@@ -799,3 +799,33 @@ def pre_upgrade_system_checks(capsules):
...
+    :return: custom certs
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: I dont see ```return``` anything here in this func. 

Just remove it, not required.

File: upgrade/helpers/tasks.py [hunk 4]
```diff
@@ -799,3 +799,33 @@ def pre_upgrade_system_checks(capsules):
...
+    """
+    certs_script = StringIO()
+    certs_script.write(
+        u"#! /bin/bash\n"
+        u"name={hostname}\n"
+        u"mkdir ownca\n"
+        u"pushd ownca\n"
+        u"wget https://raw.githubusercontent.com/ntkathole"
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: See if you can make it in 99 words as now we have changed that setting in .flake8

Applicable for all other lines.

**Pull request merged**